### PR TITLE
Config monitor - Cassandra and Zookeeper

### PR DESCRIPTION
**Pull request by basilsaji-ril**
Pull request for config monitoring

### CODE REVIEW

File: manifests/monitoring/vpccfg.pp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   ## Add the name of the scripts you have added in files/monitor_scripts/
   ## Example: $monitors = ['monitor_xmpp.sh','monitor_cassandra.sh']
-  $monitors=['contrail_control_introspect.sh']
+  $monitors=['contrail_control_introspect.sh','contrail_config_introspect.sh']
   rjil::monitoring { $monitors:}
```
Comment (vpramo) [inline manifests/monitoring/vpccfg.pp]: For the jpype dependency add the below
 package{'python-jpype':}

File: manifests/monitoring/vpcctrl.pp [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
   ## Add the name of the scripts you have added in files/monitor_scripts/
   ## Example: $monitors = ['monitor_xmpp.sh','monitor_cassandra.sh']
-  $monitors=['contrail_control_introspect.sh']
+  $monitors=['contrail_control_introspect.sh','contrail_config_introspect.sh']
   rjil::monitoring { $monitors:}
```
Comment (vpramo) [inline manifests/monitoring/vpcctrl.pp]: Same as above

**Comment by basilsaji-ril** [comment]
Adding the same. Kindly go ahead with the merge

**Comment by basilsaji-ril** [comment]
Added the same. Kindly go ahead with the merge

**Comment by vpramo** [comment]
Validated and verified merging PR

**Pull request merged**